sys/net/gnrc/netreg/gnrc_netreg.c

```diff
@@ -37,6 +37,9 @@ void gnrc_netreg_init(void)
 
 int gnrc_netreg_register(gnrc_nettype_t type, gnrc_netreg_entry_t *entry)
 {
+    /* only threads with a message queue are allowed to register at gnrc */
+    assert(sched_threads[sched_active_pid]->msg_array);
```

Comment:
Ah wait. Shouldn't that be `sched_active_thread` instead of `sched_threads[sched_active_pid]`?